go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.0
 	github.com/go-errors/errors v1.0.1
 	github.com/go-openapi/spec v0.19.5
+	github.com/instrumenta/kubeval v0.0.0-20201118090229-529b532b1ea1
```

Comment:
Can we depend on a proper release instead of HEAD?

Reply:
I followed the instructions here https://www.kubeval.com/go/ and this is what go imported. When I tried their release tag it doesn't work. How is it different from https://github.com/GoogleContainerTools/kpt/blob/master/go.mod#L16.

Reply:
It is not different. Just in general we should try to depend on proper releases. But if that doesn't work, then we don't have any choice.